packages/webapp/src/elections/components/ongoing-election-components/ongoing-round-segment.tsx

```diff
@@ -156,19 +157,21 @@ export const OngoingRoundSegment = ({
         isErrorChiefs ||
         isErrorMemberData ||
         isErrorCurrentMember ||
-        members?.length !== voteData?.length;
+        (voteData &&
+            voteData.length > 0 &&
+            members?.length !== voteData?.length);
 
-    if (isError || !members || !voteData) {
+    if (isError) {
         return <ErrorLoadingElection />;
     }
 
-    const userVoterStats = voteData.find(
+    const userVoterStats = voteData!.find(
         (vs) => vs.member === loggedInMember?.account
     );
 
-    const userVotingFor = members.find(
-        (m) => m.account === userVoterStats?.candidate
-    );
+    const userVotingFor =
+        voteData &&
```

Comment:
We already check for the presence of `voteData` above...I think we can drop line `173`.